rtl/eclrtl/rtldynfield.cpp

```diff
@@ -41,17 +41,16 @@ extern ECLRTL_API RecordTranslationMode getTranslationMode(const char *val, bool
     else if (strieq(val, "alwaysDisk") || strieq(val, "disk"))
     {
         if (!isLocal)
-            throw makeStringException(0, "alwaysDisk translation mode can only be set via HINT");
+            WARNLOG("alwaysDisk translation mode should only ever be used via a HINT");
         return RecordTranslationMode::AlwaysDisk;
     }
     else if (strieq(val, "alwaysECL") || strieq(val, "ecl"))
     {
         if (!isLocal)
-            throw makeStringException(0, "alwaysECL translation mode can only be set via HINT");
-        return RecordTranslationMode::AlwaysECL;
+            return RecordTranslationMode::AlwaysECL;
+        WARNLOG("alwaysECL translation mode should only ever be used via a HINT. Turning translation off.");
```

Comment:
I would word differently - "Ignoring alwaysECL translation global setting - use with HINT to set locally"

Reply:
@richardkchapman - It's not quite ignoring, to ignore completely would mean standard (payload) translation was on,
it's treating it to mean turn translation off.

How about:
```
Unsupported alwaysECL translation mode used globally, translation disabled - use with HINT to set locally.
```

Reply:
@jakesmith Works for me...